ckanext/cfpb_extrafields/plugin.py

```diff
@@ -524,3 +524,10 @@ def before_index(self, pkg_dict):
     def before_view(self, pkg_dict):
         return pkg_dict
 
+class ExportPlugin(p.SingletonPlugin):
+    p.implements(p.IRoutes, inherit=True)
+
+    def after_map(self, map):
+        map.connect("export_page", "/export", controller="ckanext.cfpb_extrafields.controllers.export:ExportController", action="index")
```

Comment:
Yep, learned a lot by reading through source code of various other ckan extensions.